google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITQueryTest.java

```diff
@@ -865,4 +865,80 @@ public void multipleInequalityFieldsWithDocumentKey() throws Exception {
     checkQuerySnapshotContainsDocuments(query3, "doc2", "doc3", "doc4");
     checkQuerySnapshotContainsDocuments(query3WithCursor, "doc2", "doc3", "doc4");
   }
+
+  @Test
+  public void testQueryPlan() throws ExecutionException, InterruptedException, TimeoutException {
```

Comment:
I find listing all the possible exceptions a little noisy. I think within the context of testing, we can simplify for readability:

```suggestion
  public void testQueryPlan() throws Exception {
```

Reply:
Agreed. I think my IDE added these 😆